src/test/resources/org/jabref/logic/exporter/ModsExportFormatTestAllFields.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<mods:modsCollection xmlns:mods="http://www.loc.gov/mods/v3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<mods:modsCollection xmlns:mods="http://www.loc.gov/mods/v3" xmlns:ns2="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.loc.gov/standards/mods/v3/mods-3-6.xsd">
```

Comment:
This is probably some intellij formatter stuff which ran against this, but I think this will result in faling test as the namespace is probably not written that way from the exporter logic.

Reply:
Actually, this is exactly the output from the exporter. I removed the strange `isSimilarTo` matcher and replaced it by a 1:1 string comparison which made these changes necessary.

Reply:
Ah okay. Strange. This was isSimliar matcher was coming from the xmlunit package.:
https://github.com/xmlunit/user-guide/wiki/CompareMatcher